README.md

```diff
@@ -58,3 +58,15 @@ Adding a line with emphasis directly under an image will style it as a caption:
 ![](image_url)
 *Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod.*
 ```
+
+## Image Guidelines
+
+All images, excluding external links, should have a retina version and a
+non-retina version stored in the same repository, with the suffix @2x
+appended onto the filename (E.g macbook.jpg and macbook@2x.jpg). The
+retina version should be twice the size of the non-retina version.
```

Comment:
We should probably point people at the style guide here so they have a reference for the image dimensions.